Sources/PodToBUILD/Lens.swift

```diff
@@ -95,6 +108,7 @@ public func .. <A, B, C>(lhs: Lens<A, B>, rhs: Lens<B, C>) -> Lens<A, C> {
  - returns: The composed lens.
  */
 infix operator >•>: MultiplicationPrecedence
+@available(*, deprecated)
 public func >•> <A, B, C>(lhs: Lens<A, B?>, rhs: Lens<B, C?>) -> Lens<A, C?> {
```

Comment:
still presence of the bullet character

Reply:
We can adopt the pointfreeco libs to still be functional while avoiding operators https://github.com/pointfreeco/swift-overture